widget/entry_internal_test.go

```diff
@@ -93,6 +93,29 @@ func TestEntry_DragSelect(t *testing.T) {
 	assert.Equal(t, "r the laz", entry.SelectedText())
 }
 
+func TestEntry_DragSelectWithScroll(t *testing.T) {
+	entry := NewEntry()
+	entry.SetText("The quick brown fox jumped over and over the lazy dog.")
+	entry.Resize(entry.MinSize())
```

Comment:
You shouldn't need to resize to minsize

Reply:
That's what I think, but I was unsure because previous test does it. I will remove it :)

Reply:
done